src/utils/transactions.rs

```diff
@@ -19,10 +23,14 @@ pub async fn fetch_rejections(
     safe_address: &str,
     nonce: u64,
 ) -> Option<Vec<String>> {
+    let info_provider = DefaultInfoProvider::new(&context);
+    let safe_info = info_provider.safe_info(safe_address).await.ok();
+    let is_legacy = is_legacy_domain_separator(safe_info);
```

Comment:
I would call this `use_legacy_domain_separator` as we do not test the donain separator and therefore `is_...` is misleading

Reply:
will rename, but we do test it!